app/components/account-overview-data/component.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import {tracked} from "@glimmer/tracking";
+import {task} from "ember-concurrency";
+import {inject as service} from '@ember/service';
+import {action} from '@ember/object';
+
+export default class AccountOverviewDataComponent extends Component {
+
+  @service store;
+
+  get accountId (){
+    return this.args.accountId;
+  }
+
+  @tracked dataInstance;
+
+  @action
+  getData(){
+    this.dataInstance = this.data.perform();
```

Comment:
You could also create @reads on account from that property so the template would be cleaner.

Reply:
Done

Reply:
Done